ga4gh/backend.py

```diff
@@ -782,21 +782,34 @@ def runSearchRequest(
         self.endProfile()
         return responseString
 
-    def runListReferenceBases(self, id_, requestArgs):
+    def runListReferenceBases(self, id_, request):
```

Comment:
It's confusing using ``request`` here to refer to the incoming json and the parsed protocol object. Perhaps ``requestJson`` as the parameter?